### PR TITLE
Add all files to coverage

### PR DESCRIPTION
**Pull request by adrigzr**
Note: this should reduce coverage A LOT. But it is real now...

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -9,11 +9,11 @@
     "start": "ember server",
     "pretest": "npm run clean-coverage",
     "test": "npm run test:node && npm run test:ember",
-    "test:node": "nyc --reporter=html --reporter=json --reporter=json-summary --reporter=lcov --report-dir coverage/node --exclude node-tests --exclude **/config/* mocha node-tests/**/*-test.js",
+    "test:node": "nyc mocha node-tests/**/*-test.js",
     "test:ember": "COVERAGE=true ember test",
     "clean-coverage": "rm -rf coverage",
     "merge-coverage": "cp coverage/ember/coverage-final.json coverage/ember.json && cp coverage/node/coverage-final.json coverage/node.json && nyc merge coverage coverage/coverage-final.json",
-    "check-coverage": "istanbul check-coverage coverage/coverage-final.json --statements 80 --functions 100 --branches 80 --lines 100",
+    "check-coverage": "istanbul check-coverage coverage/coverage-final.json --statements 40 --functions 40 --branches 40 --lines 40",
```
Comment (kcmr) [inline package.json]: Coverage is going to fail anyway, so why not to use the previous thresholds?

Comment (kcmr) [reply package.json]: Is not failing... ok

**Comment by adrigzr** [comment]
Coverage fails as expected. Setting up a new coverage baseline...

**Pull request merged**